app/src/App.tsx

```diff
@@ -23,19 +23,22 @@ import '@ionic/react/css/display.css';
 /* Theme variables */
 import './theme/variables.css';
 
+/* Components */
+import AssetTypeList from "./components/assets/AssetTypeList";
+import AssetType from "./components/assets/AssetType";
+import Asset from "./components/assets/Asset";
+
+
 const App: React.FC = () => {
   return (
     <IonApp>
       <IonReactRouter>
         <IonSplitPane contentId="main">
           <Menu />
           <IonRouterOutlet id="main">
-            <Route path="/" exact={true}>
-              <Redirect to="/page/Inbox" />
-            </Route>
-            <Route path="/page/:name" exact={true}>
-              <Page />
-            </Route>
+            <Route path="/assets/" component={AssetTypeList} exact/>
```

Comment:
For readability would it be better to have a Route component in Routes.tsx which returns the routes as something like:
```
function Routes(props) {
  return (
    <>
            <Route path="/assets/" component={AssetTypeList} exact/>
            <Route path="/assets/:type" component={AssetType} exact/>
            <Route path="/assets/:type/:asset" component={Asset} exact />
    </>
);
}
```

And then it can be imported into this location with `<Route />`. I just wonder if it'll make it easier to locate problems and issues with routes in the future. (and also it looks nicer when going through commits cause you can easily see when the routes changed in the git log cause routes.tsx has changed). Just a thought but open to discussion.

Reply:
Agreed, will be easier to view. Have moved